templates/corporate/support/sales_support_request.html

```diff
@@ -0,0 +1,72 @@
+            {{ csrf_input }}
+
+            <fieldset class="support-request">
+                <div class="input-box support-form-field">
+                    <label for="contact_sales_from" class="inline-block label-title">From</label>
+                    <div id="contact_sales_from" class="not-editable-realm-field">
+                        {{ user_email }}
+                        <br />
+                        Your message will be sent to
+                        <a href=mail:sales@zulip.com>Zulip Support</a>.
+                    </div>
+                </div>
+                <div class="input-box support-form-field">
+                    <label for="sales_support_request_form_role" class="inline-block label-title">Your role</label>
+                    <input id="sales_support_request_form_role" class="required" type="text" name="role" maxlength="{{ MAX_INPUT_LENGTH }}" required />
+                </div>
+                <div class="input-box support-form-field">
+                    <label for="sales_support_request_form_organization_name" class="inline-block label-title">Organization name</label>
+                    <input id="sales_support_request_form_organization_name" class="required" type="text" name="organization_name" maxlength="{{ MAX_INPUT_LENGTH }}" required />
+                </div>
+                <div class="input-box support-form-field">
+                    <label for="sales_support_request_form_organization_type" class="inline-block label-title">Organization type</label>
+                    <select id="sales_support_request_form_organization_type" name="organization_type" class="required">
+                        {% for org_type in ORG_TYPES %}
+                            <option
+                              value="{{ org_type.name }}"
+                              {% if org_type.id == realm_org_type %}
+                              selected
+                              {% endif %}
+                              >
+                                {{ org_type.name }}
+                            </option>
+                        {% endfor %}
+                    </select>
+                </div>
+                <div class="input-box support-form-field">
+                    <label for="sales_support_request_form_organization_website" class="inline-block label-title">Organization website</label>
+                    <input id="sales_support_request_form_organization_website" class="required" type="url" name="organization_website" required />
+                </div>
+                <div class="input-box support-form-field">
+                    <label for="sales_support_request_form_expected_user_count" class="inline-block label-title">Expected number of users (approximate range)</label>
+                    <input id="sales_support_request_form_expected_user_count" class="required" type="text" name="expected_user_count" maxlength="{{ MAX_INPUT_LENGTH }}" required />
+                </div>
```

Comment:
Is it easy to plumb through the current number of users, maybe just as an initial value? I feel like we should make it easy to not have to look that up.

Reply:
We don't check if the user is a billing admin here, so piping that data here might not be what the organization owner wants.

Reply:
Hmm, OK.